ts/Dashboard/EditMode/Menu/MenuItemBindings.ts

```diff
@@ -0,0 +1,35 @@
+import EditGlobals from '../EditGlobals.js';
```

Comment:
Licence  comment at the beginning of the file is missing

Reply:
I see that in other files it is also missing.
I added a task to asana for that